pkg/features/feature.go

```diff
@@ -5,19 +5,12 @@ import (
 	"strings"
 )
 
-type stage bool
-
-const (
-	devPreview  stage = false
-	techPreview stage = true
-)
-
 type feature struct {
 	envVar       string
 	name         string
 	defaultValue bool
-	overridable  bool
-	techPreview  stage
+	unchangeable bool
+	techPreview  bool
```

Comment:
I'm not sure about encoding stage as a boolean (even though today we only have a binary choice). I would prefer to do something like
```go
type Stage string

const (
    DEV_PREVIEW  Stage = "dev-preview"
    TECH_PREVIEW Stage = "tech-preview"
)
```

Reply:
But we do not want to keep our options open. Boolean has been chosen because of that: to limit the number of options.

Reply:
That's our plan today, but what if we change our mind in the future? That's why I would generally avoid boolean for such cases.

Reply:
If we change our mind, that would be a trivial local change. :older_man: doesn't like storing strings instead of booleans.